src/cmd/Makefile.am

```diff
@@ -81,6 +81,7 @@ flux_SOURCES = \
 	builtin/shutdown.c
 nodist_flux_SOURCES = \
 	builtin-cmds.c
+dist_bin_SCRIPTS = flux-python
```

Comment:
Just making sure we want this in `$prefix/bin`. I can't think of a problem with it, but currently the only thing we install there is the `flux` executable, so just thought I'd double check.

Reply:
I think you do, so that it is always there when people want to run flux scripts.

Reply:
Oh yeah, so they can do `#!/usr/bin/env flux-python`